doc/src/find_and_immediate.md

```diff
@@ -139,7 +139,19 @@ miopenConvolutionForwardCompileSolution(handle,
 
 ## Immediate Mode Fall Back
 
-The immediate mode is underpinned by the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html), however it may not contain every configuration of interest. Immediate mode's behavior when encountering a database miss is to fallback to a GEMM algorithm. The GEMM algorithm will handle most cases, however, if the user requires performance they should run the Find stage at least once. Fallback's `miopenConvolution*GetSolution` returns only one `miopenConvSolution_t` structure and its `time` member contains negative value. Future releases will implement a more robust heuristic based fallback, which is expected to provide better (but still non-optimal) performance.
+The immediate mode is underpinned by the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html), however it may not contain every configuration of interest.
+If Find-Db encounters a database miss it has two fallback paths it can take. Both fall back paths will return `miopenConvSolution_t` structure's that will have negative `time` members.
+
+### AI Based Heuristic Fall back
+
+Immediate mode's behavior on a database miss if MIOPEN_ENABLE_AI_HEUR is set to true, which it is by default, is to use an AI based heurisitic to pick the optimal solution. 
```

Comment:
Please clarify that `MIOPEN_ENABLE_AI_HEUR` is a CMake variable.

Reply:
Done.